### ISSUE TITLE
Remove autofocus after posting a comment on an issue

### ISSUE THREAD

**Issue by bs85** [isaacs/github#484]
When commenting on an issue, after a comment is successfully posted, the "Write" <textarea> is automatically focused.

Restoring the focus doesn't have a real upside as there aren't many situations where you'd post twice in a row.

It has the downside of hijacking the backspace "go back" functionality, forcing a click outside of the <textarea>, a tab or whatever before hitting backspace to return to the issues list (which is pretty common)

**Comment by DrewML** [comment]
Not a complete solution, but you can avoid the mouse click by hitting the `esc` key to remove focus from the textarea

**Comment by bs85** [comment]
@DrewML out of curiosity, what browser are you using?

Neither Firefox or Chrome (Ubuntu 15) seem to have this behavior

**Comment by DrewML** [comment]
@bs85 Good catch! I looked into it a bit more, and it turns out that it's a Chrome extension that's giving me that functionality.

Once I disabled [Vimium](https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb?hl=en), escape stopped removing focus from the textarea.

**Comment by bs85** [comment]
@DrewML thanks for looking into it! Vimium seems interesting, might give it a go
